HEC.FDA.Statistics/Distributions/Empirical.cs

```diff
@@ -402,21 +404,21 @@ public static Empirical StackEmpiricalDistributions(List<Empirical> empiricalDis
             int probabilitySteps = 2500;
             double[] cumulativeProbabilities = new double[probabilitySteps];
             double[] stackedInvCDFs = new double[probabilitySteps];
-          
-          
-            for ( int i = 0; i < probabilitySteps; i++)
+
+            Parallel.For(0, probabilitySteps, i =>
             {
                 double probabilityStep = (0.5 + i) / probabilitySteps;
                 double stackedValue = empiricalDistributionsForStacking[0].InverseCDF(probabilityStep);
-
                 for (int j = 1; j < empiricalDistributionsForStacking.Count; j++)
                 {
                     stackedValue = addOrSubtract(stackedValue, empiricalDistributionsForStacking[j].InverseCDF(probabilityStep));
                 }
-
                 cumulativeProbabilities[i] = probabilityStep;
                 stackedInvCDFs[i] = stackedValue;
-            }
+                Thread.Sleep(0);
```

Comment:
I'm assuming this is intentional, but it doesn't make sense to me. Maybe comment it to explain why you do it?